spec/checkout_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'checkout'
+
+describe Checkout do
+  let(:user_prompt) { "Please enter your phone number to complete the order:\n" + "+44 " }
+  let(:confirmation_prompt) { "Your order is on it's way! Please check your phone for confirmation.\n" }
+
+  describe '#place_order' do
+    it { is_expected.to respond_to(:place_order) }
+
+    it 'prompts the user for their phone number' do
+      allow(subject).to receive(:gets).and_return("1234567890")
+      allow(subject).to receive(:send_sms)
```

Comment:
It looks like you stubbed the sending of the Twilio API, nice! For more information, I recommend you check out https://github.com/makersacademy/takeaway-challenge/blob/master/docs/review.md#stubbing-the-twilio-api